apisix/core/request.lua

```diff
@@ -334,6 +335,26 @@ function _M.get_body(max_size, ctx)
 end
 
 
+function _M.get_body_table()
```

Comment:
ai-proxy PR also has this code so later we can merge from master after ai-proxy is merged.